src/components/App/index.js

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import { Router, Switch, Route } from 'react-router-dom';
 import { createBrowserHistory } from 'history';
+
+import * as S from './styles';
+import Header from '../Header';
 import HomePage from '../HomePage';
 import LoginPage from '../LoginPage';
 
 export default function App() {
     return (
         <Router history={createBrowserHistory()}>
-            <Switch>
-                <Route exact path="/" component={HomePage} />
-                <Route path="/login" component={LoginPage} />
-            </Switch>
+            <S.AppWrapper>
```

Comment:
Can name this just `S.Wrapper` - we're inside the App file so the distinction isn't really required

Reply:
good point, done